screen/contributors/src/main/java/com/ivy/contributors/IvyWalletRepositoryDataSource.kt

```diff
@@ -38,19 +39,48 @@ class IvyWalletRepositoryDataSource @Inject constructor(
 
     companion object {
         private const val CONTRIBUTORS_PER_PAGE = 100
+        private const val DISPLAY_ANONYMOUS_CONTRIBUTORS = true
+        private const val INITIAL_PAGE = 1
     }
 
-    suspend fun fetchContributors(): List<ContributorDto>? {
-        return try {
-            withContext(Dispatchers.IO) {
+    suspend fun fetchContributors(): List<ContributorDto> {
+        return withContext(Dispatchers.IO) {
```

Comment:
Can we use `dispatchersProvider.io` so the code is unit testable

Reply:
Sure!